client/src/components/Input.js

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+import COLORS from '../utils/COLORS';
```

Comment:
not sure about the all caps here, it's usually used for difficult-to-remember values that are known prior to execution. According to [javascript.info](https://javascript.info/variables#uppercase-constants). So, in my opnion, this would be correct if the colors never change. What if you want to change them later though?

Reply:
well, for the moment, i dont plan to change these primary colors, so why would i not use it as a variable? its easier to change it later though, since i only need to change the variable. Same thing as we used sass variables. thb it confused me, you see here a problem...

Reply:
nvm i missunderstood your comment